pootle/apps/pootle_project/models.py

```diff
@@ -388,7 +388,7 @@ def delete(self, *args, **kwargs):
     def clean(self):
         if self.code in RESERVED_PROJECT_CODES:
             raise ValidationError(
-                _('"%s" cannot be used as a project code' % (self.code,))
+                _('"%s" cannot be used as a project code') % (self.code,)
```

Comment:
No, it doesn't. Using `_('"%s" cannot be used as a project code', self.code)` should just work.